pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/QuarkusPostBuildAnalyzer.java

```diff
@@ -48,13 +48,13 @@ private static void postBuildCheck(String stagingPath, String productName) {
         Set<String> newDependencies;
         try {
             Document document = Jsoup.connect(stagingPathToProduct + "?C=M;O=D").get();
-            Element quarkusLatestBuild = document.select("a[href~=" + productName + "*]").first().parent().parent();
+            Elements quarkusBuilds = document.select("a[href~=" + productName + "*]");
 
-            String latest_build = quarkusLatestBuild.select("a[href]").text().replace("/", "");
-            String old_build = quarkusLatestBuild.nextElementSibling().select("a[href]").text().replace("/", "");
+            String latest_build = quarkusBuilds.first().select("a[href]").text().replace("/", "");
+            String old_build = quarkusBuilds.get(1).select("a[href]").text().replace("/", "");
 
-            log.info("Latest build is {}", latest_build);
-            log.info("Old build is {}", old_build);
+            log.info("Latest build is " + latest_build);
+            log.info("Old build is " + old_build);
```

Comment:
What is the reason for changing the logging format please?

Reply:
None, mistake on my side :) . Will revert it back to the original one